src/components/register/story.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
-import { storiesOf } from '@kadira/storybook';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions'; //eslint-disable-line
 import Register from './';
 
+const onChange = () => {
+  action('register on change');
+};
+
 storiesOf('Register')
   .add('User registration', () => (
-    <Register />
+    <Register handleChange={onChange} />
```

Comment:
Yeah, for now it's just going to silently eat it and not show it. I want to see if I can meaningfully log the input values with the addon. Need to look into it more.